vignettes/comparison.Rmd

```diff
@@ -0,0 +1,322 @@
+---
+title: Exemplary comparison of Bayesian and frequentist MMRM fits
+date: "`r format(Sys.time(), '%d/%m/%Y')`"
+output:
+  rmarkdown::html_document:
+    theme: spacelab
+    highlight: default
+    toc: yes
+    toc_float: yes
+    number_sections: true 
+vignette: >
+  %\VignetteIndexEntry{Exemplary comparison of Bayesian and frequentist MMRM fits}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+editor_options:
+  chunk_output_type: console
+  markdown:
+    wrap: 72
+---
+
+```{r setup, eval=TRUE, include=FALSE, warning=FALSE, message=FALSE}
+rm(list = ls())
+library(knitr)
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  prompt = TRUE,
+  comment = NA,
+  size = "scriptsize",
+  fig.height = 5,
+  fig.width = 6,
+  dpi = 320,
+  fig.align = "center",
+  fig.path = "figures/"
+)
+```
+
+# About {.unnumbered}
+
+This vignette provides an exemplary comparison of a Bayesian MMRM fit, 
```

Comment:
It this point, it might be good to refer readers to the section at the very bottom, where the differences between models are shown parameter by parameter.

Reply:
Agree, this has now been addressed in 6fb042aa72c1b94a959e15d186477ad83faed50a.